src/xrGame/GamePersistent.cpp

```diff
@@ -938,6 +941,9 @@ void CGamePersistent::LoadTitle(bool change_tip, shared_str map_name)
 
         pApp->LoadTitleInt(
             StringTable().translate("ls_header").c_str(), tmp.c_str(), StringTable().translate(buff).c_str());
+
+        if (map_name.c_str())
```

Comment:
```suggestion
        if (!map_name.empty())
```

Reply:
![image](https://user-images.githubusercontent.com/68949838/127742037-b847058f-c44c-44e7-a555-2383b7346bbd.png)

Reply:
Whoops, use this then:
```suggestion
        if (map_name.size())
```